src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -553,55 +553,55 @@ void KaleidoScope_DrawPages(PlayState* play, GraphicsContext* gfxCtx) {
             }
 
             if (pauseCtx->stickAdjX < -30) {
-                if (D_8082AD4C == -1) {
-                    if (--D_8082AD44 < 0) {
-                        D_8082AD44 = XREG(6);
+                if (sStickXRepeatState == -1) {
+                    if (--sStickXRepeatTimer < 0) {
+                        sStickXRepeatTimer = R_PAUSE_STICK_REPEAT_DELAY;
```

Comment:
```suggestion
                if (sStickXRepeatState == -1) {
                    sStickXRepeatTimer--;
                    if (sStickXRepeatTimer < 0) {
                        sStickXRepeatTimer = R_PAUSE_STICK_REPEAT_DELAY;
```
Just a minor thing, doesn't affect approval, but all 4 decrements of `sStickXRepeatTimer` can be moved to it's own line. I find that easier to read, but maybe others like it in one line

Reply:
I think it's better on its own line yeah, I think we've been avoiding this kind of thing